src/parser/jobs/mnk/modules/Demolish.tsx

```diff
@@ -1,8 +1,8 @@
 import {Trans} from '@lingui/react'
-import {ActionLink} from 'components/ui/DbLink'
+import {ActionLink, DataLink} from 'components/ui/DbLink'
```

Comment:
Just use DataLink exclusively?

Reply:
There were a few jobs that were using ActionLink inside a React block and spreading the action in, which DataLink wasn't working with, so I kind of just got to a point where I stopped fiddling with ActionLinks.  Fixed these to DataLink only.